web/src/beta/ui/fields/index.stories.tsx

```diff
@@ -0,0 +1,173 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import {
+  CommonField,
+  InputField,
+  TwinInputField,
+  TripletInputField,
+  QuartetInputField,
+  AssetField,
+  SelectField,
+  TextareaField,
+  SwitchField,
+  CodeField,
+  ColorField,
+  SpacingField,
+  CameraField,
+  SliderField,
+  TimePointField,
+  TimePeriodField,
+  ListField,
+  MultiSelectInputField,
+  TimeField,
+} from "./index";
+
+const meta: Meta<typeof InputField> = {
+  component: InputField,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof InputField>;
+
+const optionsMenu = [
+  {
+    id: "0",
+    title: "delete 1",
+    onClick: () => console.log("Option 1 deleted"),
+  },
+];
+
+export const Components: Story = {
+  render: () => (
+    <div style={{ display: "flex", flexDirection: "column", gap: "20px" }}>
+      <CommonField
+        commonTitle="Field Name"
```

Comment:
By defining all commonTitle as Field name it is hard to know which fields you are referring, please name it based on the component since this is shared stories

Reply:
done